velox/functions/prestosql/registration/GeneralFunctionsRegistration.cpp

```diff
@@ -48,8 +62,33 @@ void registerGeneralFunctions(const std::string& prefix) {
   VELOX_REGISTER_VECTOR_FUNCTION(udf_reduce, prefix + "reduce");
   VELOX_REGISTER_VECTOR_FUNCTION(udf_array_filter, prefix + "filter");
 
-  VELOX_REGISTER_VECTOR_FUNCTION(udf_least, prefix + "least");
-  VELOX_REGISTER_VECTOR_FUNCTION(udf_greatest, prefix + "greatest");
+  registerGreatestFunction<bool>(prefix);
```

Comment:
This is a lot of code. Perhaps, replace registerGreatestFunction with registerGreatestAndLeastFunction to reduce the size of this code in half.

Reply:
Fixed